packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -20,6 +20,27 @@ export type ConfigurationFactory = (
   args: Record<string, unknown>
 ) => Configuration | Promise<Configuration>;
 
+enum RendererTarget {
+  Web,
+  ElectronRenderer,
+  ElectronPreload,
+  SandboxedPreload,
+}
+
+function rendererTargetToWebpackTarget(target: RendererTarget): string {
+  switch (target) {
+    case RendererTarget.Web:
+    case RendererTarget.SandboxedPreload:
+      return 'web';
+    case RendererTarget.ElectronPreload:
+      return 'electron-preload';
+    case RendererTarget.ElectronRenderer:
+      return 'electron-renderer';
+    default:
+      return '';
```

Comment:
The `default` case here should not need to exist, if typescript insists it should exist instead make it `throw`